Marlin/ubl_G29.cpp

```diff
@@ -1615,22 +1615,22 @@ double zzz[] = { 0.05, 0.05, 0.15, 0.15 };
 int three_size = sizeof(xxx) / sizeof(double);
 
       results = lsf_linear_fit(xxx9, yyy9, zzz9, nine_size);
-SERIAL_PROTOCOLPAIR("\nxxx9->A =", results->A); 
-SERIAL_PROTOCOLPAIR("\nxxx9->B =", results->B); 
-SERIAL_PROTOCOLPAIR("\nxxx9->D =", results->D); 
-SERIAL_PROTOCOL("\n");
+SERIAL_ECHOPAIR("\nxxx9->A =", results->A); 
+SERIAL_ECHOPAIR("\nxxx9->B =", results->B); 
+SERIAL_ECHOPAIR("\nxxx9->D =", results->D); 
+SERIAL_ECHO("\n");
```

Comment:
`SERIAL_EOL` is preferred (prints char, not string) for single newline. `SERIAL_CHAR` for other single characters. And `SERIAL_ECHOPGM("...")` should be used for literal strings, to ensure storage in progmem.